.github/workflows/master_merge.yml

```diff
@@ -0,0 +1,103 @@
+---
```

Comment:
In current implementation, our build workflow has `workflow_dispatch` along with `pull_request` event. This means build can be triggered manually if required. 

This (workflow_dispatch) gets used today, in the label creation process to trigger the build after tag creation so that new artifact gets created and pushed.  How are we planning to achieve this with master-ci framework.

However I don't see this option with master CI.